priv/rel/files/boot

```diff
@@ -187,10 +189,10 @@ generate_sys_config() {
                 __conform="$ROOTDIR/bin/conform"
             fi
 
-            # remove old sys.config since conform seems to merge this one too
-            rm -v "$GENERATED_CONFIG_DIR/sys.config"
+            # remove old sys.config first from the mutable dir since conform also considers this file for merging
+            rm -v $SYS_CONFIG
 
-            result="$("$BINDIR/escript" "$__conform" --conf "$__conform_file" --schema "$__schema_file" --config "$SYS_CONFIG" --output-dir "$GENERATED_CONFIG_DIR")"
+            result="$("$BINDIR/escript" "$__conform" --conf "$__conform_file" --schema "$__schema_file" --config "$RELEASE_CONFIG_DIR/sys.config" --output-dir "$GENERATED_CONFIG_DIR")"
```

Comment:
outch, good catch! is that the reason why sys.config in running-config was considered/merged-in as well?